frontend/src/components/Footer/index.tsx

```diff
@@ -0,0 +1,161 @@
+    backgroundSize: 500,
+    backgroundPosition: "right center",
+    backgroundRepeat: "no-repeat",
+    opacity: 0.25,
+
+    [theme.breakpoints.down("md")]: {
+      width: "100%",
+      left: 0,
+    },
+  },
+  rdLogo: {
+    height: "100%",
+    float: "left",
+
+    [theme.breakpoints.up("md")]: {
+      marginRight: theme.spacing(4),
+    },
+
+    "&:hover": {
+      cursor: "pointer",
+    },
+  },
+  infoBox: {
+    marginTop: -80,
+    marginBottom: 72,
+  },
+}));
+
+const Footer: React.FC = () => {
+  const classes = useStyles();
+
+  const [open, setOpen] = React.useState(false);
+
+  return (
+    <>
+      <div>
+        <Box className={classes.footer}>
+          <Container>
+            <Grid container>
+              <Grid item xs={12} md={6}>
+                <Link href="/" passHref>
+                  <Typography variant="h5" className={classes.title} gutterBottom>
+                    INDØK
+                  </Typography>
+                </Link>
+                <Typography variant="body2">
+                  Foreningen for Studentene ved Industriell Økonomi og Teknologiledelse, NTNU
+                </Typography>
+                <Typography variant="body2">Kolbjørn Hejes vei 1E, 7034 Trondheim</Typography>
+                <Typography variant="body2">Org.nr. 994 778 463</Typography>
+                <a href="mailto:Leder@indokhs.no">
+                  <Typography variant="body2">Leder@indokhs.no</Typography>
+                </a>
+                <Link href="/report/">Baksida</Link>
+                <Box mt={2}>
+                  <a href="https://www.facebook.com/HovedstyretIndok" rel="noreferrer noopener" target="_blank">
+                    <IconButton edge="start" size="small" aria-label="facebook" color="inherit">
+                      <Facebook />
+                    </IconButton>
+                  </a>
+                  <a href="https://github.com/hovedstyret/indok-web" rel="noreferrer noopener" target="_blank">
+                    <IconButton size="small" aria-label="github" color="inherit">
+                      <GitHub />
+                    </IconButton>
+                  </a>
+                </Box>
+              </Grid>
+              <Grid item xs={12} md={6}>
+                <Box className={classes.nth} position="absolute" width="600px" height="100%" top={0} zIndex={-1}></Box>
+              </Grid>
+            </Grid>
+          </Container>
+        </Box>
+        <Box className={classes.credits}>
+          <Container>
+            <Box height="100%" display="flex" alignItems="center" className={classes.creditsContent}>
+              <a
+                className={classes.rdLogo}
+                href="https://github.com/hovedstyret/indok-web"
+                rel="noreferrer noopener"
+                style={{ height: "100%" }}
+              >
+                <Image src={rubberdokLogo} alt="Rubberdøk logo" width="67px" height="35px" layout="fixed" />
```

Comment:
How does the hardcoded height and width interact with different screen sizes?

Reply:
From my local testing, it works well. The fixed height and width is there to prevent the logo from scaling away to nothing when changing the window.